rom/dev/README.md

```diff
@@ -219,16 +222,25 @@ Both UDS and Field Entropy are available only during cold reset of Caliptra.
 - Keys Slot 0 - 31 are empty and Usage Bits are all cleared
 - PCR 0 - 31 are all cleared
 - Data Vault is all cleared
+- Dice Certificate mode is set to either “ECDSA-P-384 & ML-DSA-87” (aka Dual-Cert Mode) or “ECDSA-P-384 only” in the CPTRA_DICE_CERT_MODE register.
```

Comment:
nit: Let's name this `CPTRA_DICE_MODE` (drop the "cert"). Because this will also impact what DPE support is available.

Reply:
Updated.